tbf-challenge/teamwork-be
# added flagsCount to posts table

#### Pre-Submission Checklist






#### Checklist:





#### What does this PR do?


https://github.com/tbf-challenge/teamwork-be/issues/306
#### How should this be manually tested?


1. Start the server with` npm run dev`
2. In the terminal logs you will see the `flagsCount` been added to the `posts` table

##### Any background context you want to provide?



#### What are the relevant issues?
https://github.com/tbf-challenge/teamwork-be/issues/306
